lxd/resource_lxd_container.go

```diff
@@ -859,38 +860,3 @@ func resourceLxdContainerWaitForNetwork(server lxd.ContainerServer, name string)
 		return st, "NOT FOUND", nil
 	}
 }
-
-// Suppress Diff on empty name
-func suppressImageDifferences(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Would it make sense to put these in `shared.go` instead?

Reply:
We can put in in `shared.go`, or maybe have something like `instance_utils.go`, as they are currently instance specific.

What are your thoughts on that?

Reply:
Actually, now that I've thought about this some more I don't mind them located with the instance definition. So feel free to leave them as this PR has, or `instance_utils.go` sounds good to. Your decision.

Reply:
Agree, we can leave it as it is for now as there are only a few functions, and separate the logic later if needed.